src/test/resources/com/znsio/e2e/features/android/configs/config.properties

```diff
@@ -10,6 +10,7 @@ BROWSER=chrome
 ENVIRONMENT_CONFIG_FILE=./src/test/resources/environments.json
 IS_VISUAL=false
 LOG_DIR=target
+LOG_PROPERTIES_FILE= ./src/test/resources/log4j.properties
```

Comment:
Update the same in mobilab_config.properties as well

Reply:
ok